### PR TITLE
Create foo.txt

### PR DESCRIPTION
**Pull request by tomhjp-test**

### CODE REVIEW

**Comment by tomhjp-test** [comment]
Comment

**Review by tomhjp — APPROVED** [review]
LGTM

**Review by tomhjp-test — COMMENTED** [review]
Review comment

File: foo.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (tomhjp-test) [inline foo.txt:1]: Diff comment

Comment (tomhjp-test) [inline foo.txt:1]: Another diff comment

**Review by tomhjp — APPROVED** [review]
Needs changes

File: foo.txt (hunk 1 above)
Comment (tomhjp) [reply foo.txt:1]: Reply to comment

**Review by tomhjp — COMMENTED** [review]
Another review

**Review by tomhjp — CHANGES_REQUESTED** [review]
Needs changes 2

File: foo.txt (hunk 1 above)
Comment (tomhjp) [inline foo.txt:1]: Single diff comment

**Comment by tomhjp** [comment]
Main thread comment

**Pull request closed without merging**